gridmap:
  frequency: 1        # which frequency do we want to send an updated map?
  map_id: "map"      # map name
  size_x: 12           # size of x in meters
  size_y: 12           # size of y in meters
  resolution: 0.01    # size of each grid
  map_origin_x: 0     # center x position of the map
  map_origin_y: 0     # center y position of the map
  publish_topic: "/costmap_node/costmap/costmap" # The topic name of the map publisher 



obstacles:
  create_static:
    use: true
    circles:
      x_center: []
      y_center: []
      radius: []
    squares:
      x_center: [0]
      y_center: [0]
      length: [9]
      width: [7]
      orientation_deg: [0]
      line_thickness: [0.10]
  receiving_position:
    use: true
    circles:
      topics: []
      radius: []
    rectangles:
      topics: [/obstacles/rectangle2x1_1, /obstacles/rectangle2x1_2]
      lengths: [0.6, 0.6]
      widths: [1.2, 1.2]
      line_thickness: [0.05, 0.05]
    lines:
      topics: [/obstacles/line1]
      lengths: [4]
      line_thickness: [0.05]

  receiving_position_gazebo:
    use: false
    topic: "/gazebo/model_states"
    radius: 0.5
